docs/packages/pkg/Redis++.rst

```diff
@@ -0,0 +1,29 @@
+.. spelling::
+
+    Redis++
+    redis
+
+.. index:: Redis++
```

Comment:
please add the package to at least one of the already listed index-groups

https://hunter.readthedocs.io/en/latest/genindex.html